Bug: Fix make i18n-catalog command

```
make i18n-catalog
Fetching gotext
go: downloading golang.org/x/text v0.21.1-0.20250128184905-1e59086680cc
# golang.org/x/text/message/pipeline
/tmp/anax-gopath/pkg/mod/golang.org/x/text@v0.21.1-0.20250128184905-1e59086680cc/message/pipeline/extract.go:151:21: undefined: types.Unalias
/tmp/anax-gopath/pkg/mod/golang.org/x/text@v0.21.1-0.20250128184905-1e59086680cc/message/pipeline/extract.go:157:23: undefined: types.Unalias
make: *** [Makefile:604: /tmp/anax-gopath/bin/gotext] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -604,7 +604,7 @@ $(TMPGOPATH)/bin/gotext:
 	if [ ! -e "$(TMPGOPATH)/bin/gotext" ]; then \
 		echo "Fetching gotext"; \
 		export GOPATH=$(TMPGOPATH); export PATH=$(TMPGOPATH)/bin:$$PATH; \
-			go install golang.org/x/text/cmd/gotext@master; \
+			go install golang.org/x/text/cmd/gotext@latest; \
 	fi
 
 gopathlinks:

